src/components/ProfilePage/Bio.jsx

```diff
@@ -55,30 +54,56 @@ class Bio extends React.Component {
         cases={{
           editing: (
             <div role="dialog" aria-labelledby={`${formId}-label`}>
-              <Form onSubmit={this.handleSubmit}>
-                <FormGroup>
-                  <Label for={formId} id={`${formId}-label`}>
+              <form onSubmit={this.handleSubmit}>
+                {/*
+                  If we wrote our own form group component, maybe
+                  it would look like this. Encapsulates everything
+                  inside div.form-group
+
+                  <FormGroup
+                    // group info, or shared among child nodes
+                    id={formId}
+                    inline
+                    invalid={error != null}
+
+                    // input info
+                    value={bio}
+                    inputType="textarea"
+                    inputName={formId}
+
+                    // Meta/help/errors
+                    label={intl.formatMessage(messages['profile.bio.about.me'])}
+                    helpText=""
+                    errorMessage={error}
+                    validMessage={successMessage}
+
+                    // event handlers for input
+                    onInputChange={this.handleChange}
+                  />
+                */}
+                <div className="form-group">
+                  <label htmlFor={formId} id={`${formId}-label`}>
                     {intl.formatMessage(messages['profile.bio.about.me'])}
-                  </Label>
-                  <Input
+                  </label>
+                  <textarea
+                    className={`form-control ${error ? 'is-invalid' : ''}`}
                     type="textarea"
                     id={formId}
                     name={formId}
-                    value={bio || ''}
-                    invalid={error != null}
+                    value={bio}
                     onChange={this.handleChange}
                     aria-describedby={`${formId}-error-feedback`}
                   />
-                  <FormFeedback id={`${formId}-error-feedback`}>{error}</FormFeedback>
-                </FormGroup>
+                  <p className="invalid-feedback" id={`${formId}-error-feedback`}>{error}</p>
```

Comment:
One interesting thing - FormFeedback was actually obscuring that this was just a paragraph tag.